src/main/generated/options/soot/AntTask.java

```diff
@@ -370,12 +371,6 @@ public void setjava_version(String arg) {
                 || arg.equals( "8" )
                 || arg.equals( "1.9" )
                 || arg.equals( "9" )
-                || arg.equals( "1.10" )
```

Comment:
Same here. Should be added??

Reply:
This is also removed in the develop branch https://github.com/soot-oss/soot/blob/e233653f2a1bd823d20b490ff75ccc50033ff421/src/main/generated/options/soot/AntTask.java#L373

Reply:
Look at the default branch please. Also double check that you have not removed other things i added there please.

Reply:
Okay